abdiff/core/init_job.py

```diff
@@ -1 +1,36 @@
 """abdiff.core.init_job"""
+
+import logging
+import os
+
+from abdiff.config import Config
+from abdiff.core.utils import (
+    get_job_slug_and_working_directory,
+    update_or_create_job_json,
+)
+
+CONFIG = Config()
+
+logger = logging.getLogger(__name__)
+
+
+def init_job(job_name: str) -> dict:
+    """Function to initialize a new Job.
+
+    1. create a working directory for job
+    2. initialize a job.json file
+    """
+    job_slug, job_working_directory = get_job_slug_and_working_directory(job_name)
+    os.makedirs(job_working_directory)
+    logger.info(
+        f"Job '{job_slug}' initialized.  Working directory: {job_working_directory}"
+    )
+
+    job_data = {
+        "job_name": job_name,
```

Comment:
How do you picture this getting used later? It feels like the slug could possibly suffice

Reply:
That's fine to leave it in, just asking the question because I could see it becoming irrelevant in the future depending on the flow of the app. So I will re-raise if that happens!

Reply:
I thought the use of `slugify` was nifty and a nice way to manage "human-readable" job names while also supporting neat, consistent filepaths. 🤔